app/js/actions/draftLabOrderAction.js

```diff
@@ -0,0 +1,32 @@
+import {
+  ADD_DRAFT_LAB_ORDER_SUCCESS,
+  DELETE_DRAFT_LAB_ORDER_SUCCESS,
+  // DELETE_ALL_DRAFT_LAB_ORDERS_SUCCESS,
+} from './actionTypes';
+
+export const addDraftLabOrdersSuccess = orders => ({
+  type: ADD_DRAFT_LAB_ORDER_SUCCESS,
+  orders,
+});
+
+export const deleteDraftLabOrderSuccess = orders => ({
+  type: DELETE_DRAFT_LAB_ORDER_SUCCESS,
+  orders,
+});
+
+// Action to delete all the draft from the store
+// export const deleteAllDraftLabOrdersSuccess = () => ({
```

Comment:
this bit also

Reply:
The person going to implement the delete draft is going to need that...

Reply:
So they can just uncomment it

Reply:
If you wanna comment for the purpose of an action that should be attempted later, use this format;

*// TODO: add lab order here*

Reply:
Cool done that